mlflow/keras.py

```diff
@@ -234,9 +234,8 @@ def _is_tf_keras(model):
     with open(os.path.join(data_path, _KERAS_MODULE_SPEC_PATH), "w") as f:
         f.write(keras_module.__name__)
 
-    # By default, Keras uses the SavedModel format -- specified by "tf"
-    # However, we choose to align with prior default of mlflow, HDF5
-    save_format = kwargs.get("save_format", "h5")
+    # Use the SavedModel format if `save_format` is unspecified
+    save_format = kwargs.get("save_format", "tf")
```

Comment:
Just note that we call `log_model` without `save_format` in the keras autologging:

https://github.com/mlflow/mlflow/blob/ef17afd5766dff33cf2c8d963223eb7be4a61b61/mlflow/keras.py#L679